operate/webapp/src/main/java/io/camunda/operate/webapp/rest/ClientConfig.java

```diff
@@ -45,7 +47,7 @@ public String asJson() {
     canLogout = profileService.currentProfileCanLogout();
     isLoginDelegated = profileService.isLoginDelegated();
     tasklistUrl = operateProperties.getTasklistUrl();
-    resourcePermissionsEnabled = operateProperties.getIdentity().isResourcePermissionsEnabled();
```

Comment:
❓ do we have a confirmation that the new property should be used?

Reply:
Yes, it was confirmed here: https://camunda.slack.com/archives/C06UKS51QV9/p1732288942822749?thread_ts=1732099939.173189&cid=C06UKS51QV9